pkg/cmd/server/origin/handlers.go

```diff
@@ -21,27 +21,35 @@ import (
 	"github.com/openshift/origin/pkg/authorization/authorizer"
 )
 
+// RootPaths is the list of root paths we serve under "/". We expose it here so that we can
+// access it from integration tests
+var RootPaths = unversioned.RootPaths{}
+
 // TODO We would like to use the IndexHandler from k8s but we do not yet have a
 // MuxHelper to track all registered paths
-func indexAPIPaths(handler http.Handler) http.Handler {
+func indexAPIPaths(apiVersions []string, handler http.Handler) http.Handler {
+	// TODO once we have a MuxHelper we will not need to hardcode this list of paths
+	rootPaths := []string{"/api"}
+	for _, path := range apiVersions {
+		rootPaths = append(rootPaths, "/api/"+path)
+	}
+	rootPaths = append(rootPaths, []string{"/controllers",
+		"/healthz",
+		"/healthz/ping",
+		"/healthz/ready",
+		"/logs/",
+		"/metrics",
+		"/osapi",
+		"/osapi/v1beta3",
```

Comment:
Apparently not... the integration test picks it up. Should the `osapi` endpoints be generated the same way as the `api` endpoints?